### PR TITLE
chore: stop using asset types

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes APP-631

### CODE REVIEW

File: src/model/migrations.ts [hunk 1]
```diff
@@ -503,20 +503,12 @@ export default async function runMigrations() {
...
         });
 
         const hiddenCoinsMigrated = hiddenCoins.map((address: string) => {
           const asset = ethereumUtils.getAsset(assets, address);
-          if (asset?.type && isL2Asset(asset.type)) {
-            return `${asset.address}_${asset.network}`;
-          } else {
-            return address;
-          }
+          return `${asset.address}_${asset.network}`;
```
Comment (benisgold) [inline src/model/migrations.ts:511]: can use `getUniqueId` util

File: src/screens/transaction-details/components/TransactionDetailsValueAndFeeSection.tsx [hunk 2]
```diff
@@ -25,10 +25,7 @@ export const TransactionDetailsValueAndFeeSection: React.FC<Props> = ({
   transaction,
 }) => {
   const { network, symbol, type, fee } = transaction;
-  const assetUniqueId =
-    transaction.network === Network.mainnet
-      ? `${transaction.address}`
-      : `${transaction.address}_${transaction.network}`;
+  const assetUniqueId = `${transaction.address}_${transaction.network}`;
```
Comment (benisgold) [inline src/screens/transaction-details/components/TransactionDetailsValueAndFeeSection.tsx:28]: can use `getUniqueId` util

File: src/utils/ethereumUtils.ts [hunk 3]
```diff
@@ -76,10 +74,7 @@ const getNetworkNativeAsset = (
   network: Network
 ): ParsedAddressAsset | undefined => {
   const nativeAssetAddress = getNetworkObj(network).nativeCurrency.address;
-  const nativeAssetUniqueId =
-    network === Network.mainnet
-      ? nativeAssetAddress
-      : `${nativeAssetAddress}_${network}`;
+  const nativeAssetUniqueId = `${nativeAssetAddress}_${network}`;
```
Comment (benisgold) [inline src/utils/ethereumUtils.ts:77]: can use `getUniqueId` util

File: src/components/expanded-state/swap-details/SwapDetailsContent.js [hunk 4]
```diff
@@ -54,12 +54,8 @@ export default function SwapDetailsContent({
   const [detailsExpanded, setDetailsExpanded] = useState(false);
 
   const colorForAsset = useColorForAsset(outputCurrency, undefined, true, true);
-  const inputCurrencyNetwork = ethereumUtils.getNetworkFromType(
-    inputCurrency?.type
-  );
-  const outputCurrencyNetwork = ethereumUtils.getNetworkFromType(
-    outputCurrency?.type
-  );
+  const inputCurrencyNetwork = inputCurrency.network;
```
[the remark this answers is not in the document]
Comment (skylarbarrera) [reply src/components/expanded-state/swap-details/SwapDetailsContent.js:57]: this sheet is for swap review so there will have to be both currencies to get here

File: src/hooks/useSwapDerivedOutputs.ts [hunk 5]
```diff
@@ -187,17 +187,17 @@ const getOutputAmount = async (
   if (!inputAmount || isZero(inputAmount) || !outputToken) return null;
 
   try {
-    const outputChainId = ethereumUtils.getChainIdFromType(
-      outputToken?.type || AssetType.token
+    const outputChainId = ethereumUtils.getChainIdFromNetwork(
+      outputToken.network
```
[the remark this answers is not in the document]
Comment (skylarbarrera) [reply src/hooks/useSwapDerivedOutputs.ts:191]: theres a null check we gucci

File: src/hooks/useSwapRefuel.ts [hunk 6]
```diff
@@ -52,19 +52,11 @@ export default function useSwapRefuel({
...
-    const outputNetwork = ethereumUtils.getNetworkFromType(
-      outputCurrency?.type
-    );
-    const chainId =
-      inputCurrency?.type || outputCurrency?.type
-        ? ethereumUtils.getChainIdFromType(
-            inputCurrency?.type ?? outputCurrency?.type
-          )
-        : 1;
+    const inputNetwork = inputCurrency.network;
```
[the remark this answers is not in the document]
Comment (skylarbarrera) [reply src/hooks/useSwapRefuel.ts:55]: this should be fine i made network a mandatory field

[the remark this answers is not in the document]
Comment (skylarbarrera) [reply src/hooks/useSwapRefuel.ts:55]: actually im prob wrong cause they could be undefined but also just found out this isnt being used anywhere

**Review by benisgold — APPROVED** [review]

**Pull request merged**